examples/gallery/embellishments/timestamp.py

```diff
@@ -0,0 +1,18 @@
+"""
+Timestamp
+---------
+The :meth:`pygmt.Figure.timestamp` method can draw the
+GMT time stamp logo on the map/plot. A custom label
+can be added via the ``label`` parameter. The timestamp
+will always be shown in the bottom-left corner of the
+figure.
```

Comment:
I would be a bit more detailed here. Feel free, to improve my formulation.

```suggestion
can be added via the ``label`` parameter. The timestamp
will always be shown relative to the bottom-left corner
of the figure. By default, the ``offset`` and
``justification`` parameters are set to
``("-54p", "-54p")`` (x, y directions) and ``BL``
(bottom-left), respectively. 
```

Reply:
Hm. Based on @michaelgrund's comment https://github.com/GenericMappingTools/pygmt/pull/2395#discussion_r1131261305, I am wondering whether this formulation is strictly speaking also not precise / correct and "figure" should be "map" (or "plot")?

Reply:
Perhaps open a separate issue so that we can have more discussions.

Reply:
Please see issue #2437.